bps/clustering/clustering_HSC_ApTemplate.yaml

```diff
@@ -13,7 +13,7 @@
 clusterAlgorithm: lsst.ctrl.bps.quantum_clustering_funcs.dimension_clustering
 cluster:
   singleFrame:
-    pipetasks: isr,characterizeImage,calibrate
+    pipetasks: isr,calibrateImage
```

Comment:
Leave this all as is.

Reply:
`ApTemplate` [does, in fact, use `calibrateImage`](https://github.com/lsst/ap_pipe/blob/main/pipelines/_ingredients/ApTemplate.yaml#L12).